message_sender/models.py

```diff
@@ -76,3 +77,28 @@ class OutboundSendFailure(models.Model):
 
     def __str__(self):  # __unicode__ on Python 2
         return str(self.id)
+
+
+@python_2_unicode_compatible
+class Channel(models.Model):
+
+    VUMI_TYPE = 'vumi'
+    JUNEBUG_TYPE = 'junebug'
+
+    CHANNEL_TYPES = (
+        (JUNEBUG_TYPE, 'Junebug'),
+        (VUMI_TYPE, 'Vumi')
+    )
+
+    channel_id = models.CharField(unique=True, null=False, blank=False,
+                                  max_length=64)
```

Comment:
We could use this as the primary key field of the model (`primary_key=True`), to avoid having 2 unique ways to identify a channel. (django automatically adds an auto-incrementing `id` field to every model if you don't specify a primary key)